src/bindings/python/tests/test_transformations/test_graph_rewrite.py

```diff
@@ -4,7 +4,7 @@
 from openvino.runtime import opset8
 from openvino.runtime.passes import Manager, GraphRewrite, MatcherPass, WrapType, Matcher
 
-from utils.utils import count_ops, get_test_model, PatternReplacement
+from .utils.utils import count_ops, get_test_model, PatternReplacement
```

Comment:
```suggestion
from .utils.utils import count_ops, get_test_model, PatternReplacement
```
Why change all `utils` to relative `.utils`?

Reply:
I added `__init__.py` file into test_transformations to find `test_utils`.
it caused to change the way how utils is imported.